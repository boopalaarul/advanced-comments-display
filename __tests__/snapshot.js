import { render } from '@testing-library/react'
import Page from '../src/app/page'
import CommentsSection from '../src/app/ui-components/comments-section'
 
it('renders homepage unchanged', () => {
  const { container } = render(<CommentsSection />)
  expect(container).toMatchSnapshot()
})