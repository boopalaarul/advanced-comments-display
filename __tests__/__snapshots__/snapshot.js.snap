// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="bg-yellow-500 p-10 flex flex-col space-y-5"
  >
    <h1
      class="text-2xl text-bold text-black"
    >
      Comments
    </h1>
    <p
      class="text-black"
    >
      <strong>
        Comments policy:
      </strong>
       Be civil etc etc
    </p>
    <div
      class="flex flex-row space-x-5"
    >
      <p
        class="text-black"
      >
        Comments displaying in: 
        chronological
         mode.
      </p>
      <button
        class="rounded-md bg-gray-500 px-[4px]"
      >
        Change Display Mode
      </button>
    </div>
  </div>
</div>
`;
